DeepSpeech.py

```diff
@@ -1414,6 +1462,9 @@ def train(server=None):
     apply_gradient_op = optimizer.apply_gradients(avg_tower_gradients, global_step=global_step)
 
 
+    if FLAGS.early_stop is True and not FLAGS.validation_step > 0:
+        log_warn('Validation flag needs to be enabled for Early Stopping to work')
```

Comment:
How about
`log_warn('Parameter --validation_step needs to be >0 for early stopping to work')`?